samples/world-clock/presenters/src/androidMain/kotlin/app/cash/zipline/samples/worldclock/android.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2022 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.samples.worldclock
+
+import android.content.Context
+import app.cash.zipline.loader.ManifestVerifier.Companion.NO_SIGNATURE_CHECKS
+import app.cash.zipline.loader.ZiplineLoader
+import java.util.concurrent.Executors
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.cancel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.flowOf
+import okhttp3.OkHttpClient
+
+class WorldClockAndroid(
+  private val applicationContext: Context
+) {
+  private val ziplineExecutorService = Executors.newSingleThreadExecutor { Thread(it, "Zipline") }
+  private val ziplineDispatcher = ziplineExecutorService.asCoroutineDispatcher()
+  private val scope = MainScope()
```

Comment:
This scope should really be owned by the activity since it's the thing providing the lifecycle.

Reply:
Can fix. Follow-up!